drawBot/context/baseContext.py

```diff
@@ -403,14 +412,30 @@ def _getCGPath(self):
             elif instruction == AppKit.NSClosePathBezierPathElement:
                 Quartz.CGPathCloseSubpath(path)
         # hacking to get a proper close path at the end of the path
-        x, y, _, _ = self.bounds()
-        Quartz.CGPathMoveToPoint(path, None, x, y)
-        Quartz.CGPathAddLineToPoint(path, None, x, y)
-        Quartz.CGPathAddLineToPoint(path, None, x, y)
-        Quartz.CGPathAddLineToPoint(path, None, x, y)
-        Quartz.CGPathCloseSubpath(path)
+        if withBounds:
```

Comment:
It's not drawing bounds, really, just adding the same point several times over. I don't understand the purpose, and I hope it can be deleted. @typemytype?

Reply:
Ive I remember it correctly a text setter needs a proper closed path... will test again...

Reply:
Just could you run the test without adding that single point?

it was added here https://github.com/typemytype/drawbot/commit/9f48f99bfc0dffa70ab759a42a2ea5fa70f985ab
while working on the hyphenation...

Reply:
I leave it up to you ( ͡~ ͜ʖ ͡°)

Reply:
The tests run fine without that section.

Reply:
So the whole `withBounds` parameter thing can go.

Reply:
Just pushed!